src/main/java/oidc/endpoints/JwkKeysEndpoint.java

```diff
@@ -4,7 +4,7 @@
 import com.google.crypto.tink.CleartextKeysetHandle;
 import com.google.crypto.tink.JsonKeysetWriter;
 import com.google.crypto.tink.KeysetHandle;
-import com.google.crypto.tink.aead.AeadKeyTemplates;
+import com.google.crypto.tink.aead.AesCtrHmacAeadKeyManager;
```

Comment:
Why do we need to change the crypto stuff for the consent feature?

Reply:
The methods we used were deprecated. This is the preferred way now. There is very good test coverage for this and it did not break anything